src/api/Metadata.js

```diff
@@ -705,15 +709,28 @@ class Metadata extends File {
                 const cache: APICache = this.getCache();
                 const key = this.getMetadataCacheKey(id);
                 const cachedMetadata = cache.get(key);
-                const editor = this.createEditor(metadata.data, template, canEdit);
-                if (cachedMetadata && cachedMetadata.editors) {
-                    cachedMetadata.editors.splice(
-                        cachedMetadata.editors.findIndex(({ instance }) => instance.id === editor.instance.id),
-                        1,
-                        editor,
-                    );
+
+                if (isMetadataRedesign) {
+                    const templateInstance = this.createTemplateInstance(metadata.data, template, canEdit);
```

Comment:
few lines above, the update api call is triggered - I don't see a reason why to trigger create an instance for redesigned version 🤔